Source/Core/DolphinQt/RenderWidget.cpp

```diff
@@ -37,6 +37,7 @@
 #include "VideoCommon/VideoConfig.h"
 
 #ifdef _WIN32
+#include <Windows.h>
 #include <WinUser.h>
```

Comment:
Could you remove these two lines? As @Pokechu22 said, including `<Windows.h>` [automatically includes](https://en.wikipedia.org/wiki/Windows.h#Child_header_files) winuser.h and windef.h.

Reply:
Fixed.